README.md

```
@@ -1,17 +1,181 @@
+
+<details>
+<summary>Click to expand deprecated usage documentation</summary>
+
+### AWS CDK version
 
 Now using `v1.47.0`.
 
-## Install
+### Install
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)